### PR TITLE
When I am getting a new client, I want to be able to add that client

### PR DESCRIPTION
**Pull request by jellebouwman**
Added a new page, with largely the same structure and design as the recently built TeamMembers page. The datastructure and formfields are somewhat different, to accommodate for a more realistic information collection.

Clients are presented, can be sorted and a new client can be added on a seperate form.

### CODE REVIEW

File: components/client/Client.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          >
+            <img
+              alt="arrow down sign"
+              className="client__expand-button-icon"
+              src="/static/icons/arrow-down-2.svg"
+            />
+          </button>
+        </div>
+        <div className={`
+          client__expand
```
Comment (RikFrieling) [inline components/client/Client.jsx]: expand is a verb, so it implies an action. Maybe you could find a noun as an element name?

Comment (jellebouwman) [reply components/client/Client.jsx]: I changed it to `__detail` in the last commit!

File: components/add-client/AddClient.jsx [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+  handleSubmit = (event) => {
+    const { client } = this.state;
+    event.preventDefault();
+
+    const newClient = {
+      ...client,
+      avatar: 'static/icons/humanoids-avatar.jpg'
+    };
+    this.handleAddClient(newClient);
```
Comment (boris-arkenaar) [inline components/add-client/AddClient.jsx]: You could just call `this.props.onPostClient(newClient)` directly here. `handleAddClient` isn't used anywhere else I believe.

File: components/add-client/add-client.scss [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+    font-weight: 600;
+    height: 36px;
+    margin-top: 7px;
+    padding: 0 0 0 13px;
+    width: 100%;
+  }
+
+  &__input-contact-information {
+    border: solid 1px $color-accent-alt-9;
+    border-bottom-left-radius: 0;
```
Comment (boris-arkenaar) [inline components/add-client/add-client.scss]: You could write `border-radius: 0 4px 4px 0;`

File: components/client-overview/ClientOverview.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            <option value="descending">
+              Descending
+            </option>
+          </select>
+        </section>
+        <ul className="client-overview__content-wrapper">
+          {clients.map(client => (
+            <li key={client.id}>
+              <Client
+                avatar={client.avatar}
```
Comment (boris-arkenaar) [inline components/client-overview/ClientOverview.jsx]: You could write `...{client}`.

File: components/client/Client.jsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    phoneNumber: PropTypes.string.isRequired,
+    remarks: PropTypes.string.isRequired,
+    webAddress: PropTypes.string.isRequired
+  }
+
+  state = {
+    isClientExpanded: false
+  }
+
+  toggleExpand = () => {
```
Comment (boris-arkenaar) [inline components/client/Client.jsx]: Name this `toggleDetail` as well.

File: ducks/clients.js [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+      return 0;
+    })
+  )
+);
+
+
+export function clientReducer(state = initialState, action) {
+  switch (action.type) {
+    case REQUEST_CLIENTS:
+      return { ...state };
```
Comment (boris-arkenaar) [inline ducks/clients.js]: Not setting `isLoading` to `true`?

**Comment by boris-arkenaar** [comment]
I don't yet see the option to delete a client.

**Comment by jellebouwman** [comment]
True - deleting a client is not included, but this was not part of the story!

File: components/client-overview/ClientOverview.jsx (hunk 4 above)
Comment (jellebouwman) [reply components/client-overview/ClientOverview.jsx]: I am passing some information through that I'm not using yet - such as `zipCode` and `address`, I agree this is better when I'm passing through the entire object.

**Pull request merged**